patches/chrome-android-java-src-org-chromium-chrome-browser-sync-AndroidSyncSettings.java.patch

```diff
@@ -1,13 +1,13 @@
 diff --git a/chrome/android/java/src/org/chromium/chrome/browser/sync/AndroidSyncSettings.java b/chrome/android/java/src/org/chromium/chrome/browser/sync/AndroidSyncSettings.java
-index 09c8dfc92481b272b536fdc05dfb7b11fbca616d..c7e325ac1062731e389059ce522045fcaf9b4e44 100644
+index 09c8dfc92481b272b536fdc05dfb7b11fbca616d..629b7c451cba0e9f7d0f5c25cf561b76221a68d9 100644
 --- a/chrome/android/java/src/org/chromium/chrome/browser/sync/AndroidSyncSettings.java
 +++ b/chrome/android/java/src/org/chromium/chrome/browser/sync/AndroidSyncSettings.java
 @@ -74,7 +74,7 @@ public class AndroidSyncSettings {
      public static AndroidSyncSettings get() {
          ThreadUtils.assertOnUiThread();
          if (sInstance == null) {
 -            sInstance = new AndroidSyncSettings(getSyncAccount());
-+            sInstance = new BraveAndroidSyncSettings(contentResolver);
++            sInstance = new BraveAndroidSyncSettings(getSyncAccount());
```

Comment:
seems like this is a candidate for asm now since it takes the same parameter?

Reply:
It requires additional fix in bytecode, issue created https://github.com/brave/brave-browser/issues/12284